src/common/utils/pagination.ts

```diff
@@ -0,0 +1,41 @@
+import { DeserializedList, PaginationOptions} from '../interfaces';
+// Autopaginatable class definition
```

Comment:
We don't need this comment. It's basically just a rephrasing of the next line :)